package.json

```diff
@@ -4,7 +4,7 @@
     "description": "A Chrome Extension that listens for dice rolls on D&D Beyond Character Sheets (e.g. [Remigius](https://www.dndbeyond.com/profile/kamnikite/characters/23934295)) and posts the results into a Discord server via a bot.",
     "main": "content.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "dev:options": "parcel watch options/index.html --no-source-maps --public-url /build_output --out-dir ./build_output --out-file options.html --hmr-port 8080"
```

Comment:
Giant command that watches the file system for changes and rebuilds.

Documentation here: https://parceljs.org/cli.html